src/ring_theory/norm.lean

```diff
@@ -225,6 +225,11 @@ end
 
 variable (K)
 
+/--For `L/K` a finite separable extension of fields and `E` an algebraically closed extension
+of `K`, the norm (down to `K`) of an element `x` of `L` is equal to the product of the images
+of `x` over all the `K`-embeddings `σ`  of `L` into `E`.
+ -/
+
```

Comment:
I'm not sure what the preferred style is, but this looks better to my eyes.
```suggestion
/-- For `L/K` a finite separable extension of fields and `E` an algebraically closed extension
of `K`, the norm (down to `K`) of an element `x` of `L` is equal to the product of the images
of `x` over all the `K`-embeddings `σ`  of `L` into `E`. -/
```

Reply:
sure!